### ISSUE TITLE
Unhandled Exception

### ISSUE THREAD

**Issue by jackbuilder52** [piratesephiroth/TeconmoonWiiVCInjector#2]
It worked fine for the first few days on the same ISO (Super Smash Bros Melee). i even played it on my wii u. but i had to delete it and now im installing it again. but this time, it's not working. i have tried downloading and redownloading but no luck.

![image](https://user-images.githubusercontent.com/49613637/127551459-d3281732-9b5a-40fa-ba34-e4fb196b801b.png)

**Comment by piratesephiroth** [comment]
Hmm, that's weird. It seems to have had an error after the conversion was already finished.
However that path is weird... did you select that as the place to save the resulting installable files?

What does it say when you press the "Details" button?

**Comment by jackbuilder52** [comment]
No. The path i selected to save the output files is on the root of Drive F, one of my internal drives. First, i tried selecting my SD Card but the same error popped up. Here's what's under the Details tab [https://pastebin.com/CiUqxWC2](url) (Pastebin Link for organization and easier viewing.

**Comment by piratesephiroth** [comment]
Well that was a weird error, it didn't find nuspacker's temporary folders. Since I'm not sure of what caused it (I suppose nuspacker ran fine), I just made a small change so now it only tries to delete those folders if they exist.

Try the most recent release,
